package.json

```diff
@@ -158,11 +153,6 @@
                 "title": "%appService.EnableFileLogging%",
                 "category": "Azure App Service"
             },
-            {
-                "command": "appService.InstallCosmosDBExtension",
```

Comment:
I don't think we want to include removing all the cosmos connection stuff in this PR as well.

Reply:
Oh maybe I forgot to merge with main